src/data/zsqrtd/basic.lean

```diff
@@ -277,6 +277,7 @@ cast nonnegg_comm (nonnegg_cases_right h)
 section norm
 
 def norm (n : ℤ√d) : ℤ := n.re * n.re - d * n.im * n.im
+lemma norm_def (n : ℤ√d) : n.norm = n.re * n.re - d * n.im * n.im := rfl
```

Comment:
I think this lemma already exists as `norm` - that is `rw norm_def` is just a longer spelling of `rw norm`.